spec/eof.md

```diff
@@ -214,6 +214,7 @@ The following instructions are introduced in EOF code:
     - pops `value`, `salt`, `data_offset`, `data_size` from the stack
     - load initcode EOF subcontainer at `initcontainer_index` in the container from which `EOFCREATE` is executed
     - deduct `6 * ((initcontainer_size + 31) // 32)` gas (hashing charge)
+    - check call depth limit and whether caller balance is enough to transfer `value`
```

Comment:
Actually call depth check is removed for new CALLs in EIP-7069, I think we should drop it here, too.